src/containers/Tenant/Query/ExplainResult/components/SimplifiedPlan/OperationParams.tsx

```diff
@@ -43,25 +43,18 @@ function reorderParamsEntries(params: Record<string, unknown>) {
 
 function getOperationParams(params: Record<string, unknown> = {}) {
     const result = [];
-    const paramsEntries = Object.entries(params);
-    if (paramsEntries.length === 1) {
```

Comment:
Why did you decide to remove this logic? As discussed, if operation has only one parameter, it should be displayed without key.

Reply:
reverted